### ISSUE TITLE
parca: `\"type\" already defined` error when using flux

### ISSUE THREAD

**Issue by adomenech73** [rlex/helm-charts#23]
Hi

Perhaps I'm missing something but could it be possible that this line is redundant?: https://github.com/rlex/helm-charts/blob/f415d9d88c82ede76041a4bc7c675c10deac178c/charts/parca/templates/agent-service.yaml#L16

**Comment by rlex** [comment]
Good find! I'm not using agent so missed that.
I released 1.5.0 both bumping parca version and fixing that issue.

**Comment by fmendo** [comment]
Hi there,

I found other duplicated variable definition in [agent-daemonset.yaml](https://github.com/rlex/helm-charts/blob/master/charts/parca/templates/agent-daemonset.yaml) template file:

Lines 26 and 93: `serviceAccountName: {{ include "parca.serviceAccountName" . }}`

Please have a look, thanks! :)

**Comment by rlex** [comment]
1.5.1 should fix that, thanks again!

**Comment by fmendo** [comment]
I found last one, same file [agent-daemonset.yaml](https://github.com/rlex/helm-charts/blob/master/charts/parca/templates/agent-daemonset.yaml) template file:

Lines 31-32 and 57-58:
```
          securityContext:
            {{- toYaml .Values.agent.securityContext | nindent 12 }}
```

Thanks again

**Comment by rlex** [comment]
should be fixed in 1.5.2

**Comment by rlex** [comment]
everything ok now?
